bmp280/dataPoints.json

```diff
@@ -0,0 +1,34 @@
+{
+  "dataPoints": [
+    {
+      "name": "Altitude",
+      "sensor": "bmp280",
+      "impl": "78afeb33-03cb-4d10-9457-16666f03451b",
+      "template": "TFLOAT16",
+      "precision": 2,
+      "minValue": -500,
+      "maxValue": 9000,
+      "unit": "m"
+    },
+    {
+      "name": "Pressure",
+      "sensor": "bmp280",
+      "impl": "3e9c8967-7128-45de-a8c5-ded86b239124",
+      "template": "TFLOAT16",
+      "precision": 2,
```

Comment:
I found the following values in the datasheet, which can be misleading at times. Are you basing the precision on what the library offers?

![image](https://github.com/user-attachments/assets/39b850bd-19bf-4e48-a430-29393fa40093)

Reply:
Yeah I was following Adafruit's library - https://github.com/adafruit/Adafruit_BMP280_Library